modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldValueRendererAccessorTest.java

```diff
@@ -54,46 +54,15 @@ public void testRenderMultipleValues() throws Exception {
 
 		int numberOfOptions = 2;
```

Comment:
@leticia-maciel I think we can remove this and the `_createOptionsValuesJSONArray` method and simplify by doing: 
```
DDMFormValuesTestUtil.createDDMFormFieldValue(
	"Select",
	new UnlocalizedValue(
		JSONUtil.putAll(
			"value 1", "value 2", "value 3"
		).toString()));
```